core/.gitignore

```diff
@@ -1 +1,2 @@
 /build
+.cxx/
```

Comment:
What's this for?

Reply:
I using AS 3.5, automatically generate this folder.

Reply:
What are the steps to reproduce this issue?

Reply:
AS 3.5 NDK 20.0.5392854-beta2
[.cxx dir](https://i.loli.net/2019/04/25/5cc1d2e734918.png)

Reply:
Okay. I'll check again when NDK r20 goes stable.